hw8/Task_9.ts

```diff
@@ -0,0 +1,10 @@
+// Task 9
+// Используя коллекцию **`Set`** создайте массив неповторяющийхся значений:
+//     ```javascript
+//     const numbers = [1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3,
+//     5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]
+// ```
+//     > Реализуйте решение двумя способами, используя `function declaration` & `arrow function`.
+
+const mySet = new Set ([1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3, 5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]);
+console.log([...mySet])
```

Comment:
Please slightly adjust to make it more readable
const numbers = [1, 6, 3, 2, 7, 325, 234, 54, 586, 23, 24, 64, 56, 3, 5, 7, 8, 1, 6, 8, 9, 4, 5, 61, 2, 4, 5, 6, 3, 5]
const mySet = new Set(numbers);
console.log([...mySet])

Reply:
Done.